microprofile.jdt/com.redhat.microprofile.jdt.test/src/main/java/com/redhat/microprofile/jdt/core/BasePropertiesManagerTest.java

```diff
@@ -145,8 +168,8 @@ public void run(IProgressMonitor monitor) throws CoreException {
 		return javaProject;
 	}
 
-	private static File copyProjectToWorkingDirectory(String projectName) throws IOException {
-		File from = new File("projects/maven/" + projectName);
+	private static File copyProjectToWorkingDirectory(String projectName,  String parentDirName) throws IOException {
+		File from = new File("projects/" + parentDirName+ "/" + projectName);
 		File to = new File(JavaUtils.getWorkingProjectDirectory(), projectName);

```

Comment:
I think it should be 

`File to = new File(JavaUtils.getWorkingProjectDirectory(), parentDirName + "/" + projectName);`

because a gradle and maven project could have the same name.